### ISSUE TITLE
About the rho term

### ISSUE THREAD

**Issue by woduck** [LinjieLyu/DPI#5]
I am currently reviewing your DPI code and paper, and I have encountered a parameter defined as rho=1/Sigma ^ 2.
Can you provide some insights on the term roh? Specifically, I would like to know how to determine or calculate this rho=1/Sigma ^ 2.
﻿
The rho term I understand now is the reciprocal of sigma ^ 2, and this sigma is the variance of the Gaussian distribution. In this paper, it should be the variance of the distribution of environment map.
﻿
Then the distribution is unknown to us. So we should only guess the distribution and adjust the super parameter, right?
﻿
And then what I want to know is how to guess this distribution and how to adjust this hyperparameter(the rho term)?
﻿
thank you

**Comment by LinjieLyu** [comment]
Hi. According to the original DPS paper, Appendix C.2, a good approximation of rho is between [0.1,1.0]. However, I would say this also depends on your loss function and the magnitude of the gradients. You can play this rho a little bit given different tasks empirically and decide how much guidance you want to add.
